### DISCUSSION TITLE
Quarto saying certain commands not recognized that are in the doc - PDF rendering

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#5649] [Q&A] [answered]
- I have tinytex installed. 
- running in RStudio/Windows
- when trying generate a PDF format I get: 

"running xelatex - 1

No TeX installation was detected.

Please run 'quarto install tool tinytex' to install TinyTex.
If you prefer, you may install TexLive or another TeX distribution."

- do a "list tools" and tinytex is there and says up to date when I run the list command.
- so, I try "quarto uninstall tinytex" and it says "unknown command 'uninstall'"
- try "quarto install tinytex --update-path" and it says "unknown option "update"

So RE the last 2 command errors, why are these "unknown" when they are listed commands in the guide?

**Comment by mcanouil** [comment]
What's the question/issue exactly?

Could you try to format your post using code blocks for code and write complete sentences?

**Reply by IceManTBM3** [reply]
Added context/Q, but essentially for these 2 commands listed in the doc (https://quarto.org/docs/output-formats/pdf-engine.html#managing-tex)

 "quarto uninstall tinytex" 
"quarto install tinytex --update-path" 

I get  "unknown command" and "unknown option," so a) am I doing something wrong (I don't think so), b) is there a doc bug (if so what's the write command) or is this a quarto bug (and is there a workaround)?

**Reply by mcanouil** [reply]
What version of Quarto are you using exactly?
WHat's the output of `quarto check`?

**Reply by IceManTBM3** [reply]
Thx

here it is

tmeek@AC-LT-TMEE-01 MINGW64 /c/Doc-Testing/diit-documentation/quarto (Tom_Doc_2)
$ quarto check

[>] Checking Quarto installation......OK
      Version: 1.1.251
      Path: C:\Users\tmeek\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.9.7 (Conda)
      Path: C:/ProgramData/Anaconda3/python.exe
      Jupyter: 4.8.1
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: C:/Users/tmeek/AppData/Local/Programs/R/R-4.2.2
      LibPaths:
        - C:/Doc-Testing/diit-documentation/quarto/renv/library/R-4.2/x86_64-w64-mingw32
        - C:/Users/tmeek/AppData/Local/Temp/Rtmpm6ngjC/renv-system-library
      rmarkdown: 2.14

[>] Checking Knitr engine render......OKT_Data_Dictionary_v2.xlsx

**Reply by mcanouil** [reply, marked as answer]
Consider upgrading your Quarto version (1.1.251) to the latest stable version (1.3.353) and you'll see that the commands work.

**Reply by IceManTBM3** [reply]
cool what is the upgrade command?? I'm not seeing it in the doc. I'm guessing the answer is to DLoad 1.3 and install from the ground up

**Reply by cscheid** [reply]
That's correct.

**Reply by IceManTBM3** [reply]
Ok, so yeah that upgrade worked. I'd suggest to adding a note to the doc on that so as to not confuse the user... I mean it's obvious to deduce, but.... upgrade notices would be great too

Thanks much
